### DISCUSSION TITLE
Hide Editor Tooltips - Please

### DISCUSSION THREAD

**Discussion by RonR-WebDesign** [elementor/elementor#17479] [Developers Community]
I would really like the option of turning off the overly used tooltips on the widget controls.   Like the Advanced > Margin > "Link values together".   They just get in the way... !

I tried css but that doesn't work.  They must be hard coded somewhere.

**Comment by RonR-WebDesign** [comment]
It appears that this could be done using the css attribute:  [data-tooltip].  Just can't figure out how to do this to the Elementor code.

**Reply by n0tnamed** [reply]
As they say, wait three years for what was promised, so... Most likely the issue has already been somehow resolved, but I will leave this here for those who may have the same problem.

In my case, this is not a desire, but really a problem. Sometimes when a tooltip pops up, it appears and then disappears. It immediately appeared and disappeared again, and so on again and again, and so on in a circle. CPU load 60%. The browser freezes. I have to close and open all the windows again. And so on several times a day at an arbitrary moment when a tooltip is displayed. Not every time.

It’s good that all changes are usually saved, but in general this behavior is quite... annoying.

The **data-tooltip** attribute usually hangs on **.tooltip-target** elements, which, when loading Elementor styleguide&editor scripts, are saved separately and after that any manipulation with attributes or classes does not remove the tooltip. It just show the text "null" on popup. The code **/elementor/assets/lib/tipsy/tipsy.min.js** is responsible for displaying the tooltips themselves. The code was written about (maybe more) 10 years ago and most likely has not been updated since it was included in the plugin. In any case, the current version of Elementor 3.17.1 still contains version tipsy 1.0. (I don’t know what’s with the interface in the current version, maybe in 2 years somewhere the display of tooltips will be configured, but in my version this is not yet available or is difficult to find).

In general, this is no longer a feature in my case, but a bug. Moreover, on some systems it is, on others it is not. It can take a very long time to wait for corrections of something that is difficult to confirm. And changes to the site need to be made now. What can be done. You can (1) turn off tooltips completely. You can (2) set the interval for displaying them (the default is instant display).

1) In the **tipsy.js** file, change line 22: **this.enabled = true;** on **this.enabled = false;** Save, minimize and replace the **tipsy.min.js** file. There will be no more tooltips. At all. It seems that only Elementor admin interface is affected.
2) In the **tipsy.js** file, change line 189: **delayIn: 0,** to **delayIn: 3000,** which will set the delay when displaying the tooltip to 3 seconds. Save, minimize and replace the **tipsy.min.js** file.

Since these changes are in the plugins folder, **when you update Elementor, changes will be lost**. But in extreme cases, it can help during long-term on-site work changes. Ideally, of course, the _delayIn_ script parameter should be configured in the Elementor interface like the feature you asked for 2 years ago. Some people need tips everywhere and now. And others, if they need them, they will wait 1-2-3-5-N seconds until they appear.

I hope this was appropriate.

For Google Search:
**Elementor tooltips loop**
**Elementor freezes hangs browser when displaying tooltips**
